### DISCUSSION TITLE
Reading a scene from three.js

### DISCUSSION THREAD

**Discussion by HIMANSHU6001** [donmccurdy/glTF-Transform#1464] [Q&A]
I want to apply draco compression on the scene which is currently present in the scene without downloading. I know we can read a glb or gltf file from local. I want to read the scene currently present.

**Comment by donmccurdy** [comment]
glTF Transform takes a glTF file as input, and produces a glTF file as output, so you would need to use [THREE.GLTFExporter](https://threejs.org/docs/?q=gltfex#examples/en/exporters/GLTFExporter) to export the current three.js scene as a glTF file, compress then compress that glTF file with glTF Transform. 

Note that the compressed file must be decompressed before rendering in three.js.
